update: cit-web readme (#696)
bcgov/CIT: cit3.0-web/README.md

# Frontend Application

> This frontend application contains two high-lelvel business objectives: Community Investment Opportunities Tool, and Community Information Tool.

Reviewer: Small typo
```suggestion
This frontend application contains two high-level business objectives: Community Investment Opportunities Tool, and Community Information Tool.
```

## Community Investment Opportunities Tool

> The Community Investment Opportunity Tool is a place where Economic Development Officers (EDO) can go to post possible investment opportunities for their respective communities. This tool includes a user authenticated portion where EDO's can go to see their currently posted investment opportunities as well as post new ones. It also includes a public facing view where possible investors can go to see all the currently available opportunities in the province or narrow down the opportunities on a set of filterable criteria.

Reviewer 2: province _and_ narrrow rather than or?

## Community Information Tool

> Is an emedded PowerBI report in addition to a splash page and web interface. The majority of the functionality is implemented in the PowerBI environment.

Reviewer: Small typo and wording suggestion
```suggestion
This is an embedded PowerBI report in addition to a splash page and web interface. The majority of the functionality is implemented in the PowerBI environment.
```

### `yarn lint:fix`

Runs ESLint on all files and automatically fixes any errors found.

> 

Reviewer 2: Love this section!  Super helpful!  Nice work :-)

Reviewer: I agree, giving a quick explanation of the folders to give people an idea of what to expect before they even look at any code is a great idea.

## Source Folder structure

`mocks` - scripts to mock objects in unit tests.

`components` - visual components and pages in the application.

`constants` - constants strings and numbers shared across application components.

`contexts` - context like authorization, shared across application components.

> `hooks` - components-specific code chunk, matched react life cycles well.

Reviewer: I had to read this over a couple of times, maybe it should be reworded a little bit? I might not be reading it correctly.
```suggestion
`hooks` - component-specific code chunks, intended to match react life cycles.
```
